### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#444] [Esami & Esoneri M-Z]
### Traccia

<img width="708" alt="image" src="https://github.com/user-attachments/assets/0af3a0c1-5438-40fb-9e63-9117c07c6ca9">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/6711e920-8e0b-4cc2-9016-e3d28311feae)
